app/api/domains/api.py

```diff
@@ -121,7 +121,7 @@ async def api_search(
     # assign placeholders (bindparams()) to these queries,
     # and this will let us bind those safely and efficiently.
     results_query = text(
```

Comment:
You also don't need this whole query at all, you can just all len() on the resulting list of users.

Reply:
len() is slower by 50+ms on every query if i used it. 

count() is more efficient because it returns the number of rows in the condition specified without having to retrieve all of the rows after that query is done, and then count those rows.

Reply:
> len() is slower by 50+ms on every query if i used it. 
> 
> count() is more efficient because it returns the number of rows in the condition specified without having to retrieve all of the rows after that query is done, and then counting those rows.

idk what you're talking about, I ran a timeit test on `len([1, 2, 3] * 100)` and got the result of ```2000000 loops, best of 5: 6.26 usec per loop```

Reply:
the implementation of len() for this wouldn't work on my machine and only returned no values every time, idk what it thinks its doing lol